### PR TITLE
Added BusSchedule, BusRouteAssignments, and BusReservations entities

### PR DESCRIPTION
**Pull request by rinesakuci**

### CODE REVIEW

**Review by oltahasimja — DISMISSED** [review]

File: pentasharp/Models/Entities/BusReservations.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using pentasharp.Models.Enums;
+
+namespace pentasharp.Models.Entities
+{
+    public class BusReservations
+    {
```
Comment (GentaSL) [inline pentasharp/Models/Entities/BusReservations.cs:11]: add xml summaries https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags

File: pentasharp/Models/Entities/BusRouteAssignments.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace pentasharp.Models.Entities
+{
+    public class BusRouteAssignments
+    {
```
Comment (GentaSL) [inline pentasharp/Models/Entities/BusRouteAssignments.cs:10]: add xml summaries https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags

File: pentasharp/Models/Entities/BusRoutes.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace pentasharp.Models.Entities
+{
+    public class BusRoutes
+    {
```
Comment (GentaSL) [inline pentasharp/Models/Entities/BusRoutes.cs:9]: add xml summaries https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags

File: pentasharp/Models/Entities/BusSchedule.cs [hunk 4]
```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using pentasharp.Models.Enums;
+
+namespace pentasharp.Models.Entities
+{
+    public class BusSchedule
+    {
```
Comment (GentaSL) [inline pentasharp/Models/Entities/BusSchedule.cs:11]: add xml summaries https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags

File: pentasharp/Models/Entities/Buses.cs [hunk 5]
```diff
@@ -0,0 +1,10 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace pentasharp.Models.Entities
+{
+    public class Buses
+    {
```
Comment (GentaSL) [inline pentasharp/Models/Entities/Buses.cs:11]: add xml summaries https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags

**Review by GentaSL — CHANGES_REQUESTED** [review]
add missing summaries as required

**Review by GentaSL — DISMISSED** [review]

**Review by GentaSL — APPROVED** [review]

**Review by oltahasimja — APPROVED** [review]

**Pull request merged**